src/main.ts

```diff
@@ -13,6 +13,18 @@ import LoaderDialog from "@/components/LoaderDialog.vue";
 import DeletionDialog from "@/components/DeletionDialog.vue";
 import FileUpload from "@/components/FileUpload.vue";
 import { initFromStorage } from "@/utils/startup";
+import * as Sentry from '@sentry/browser';
+import * as Integrations from '@sentry/integrations';
+
+Sentry.init({
```

Comment:
Bearing your comment in mind, I think it's important to see errors in development and we don't want Sentry notifications for development anyway (at least not in the frontend). So I suggest to wrap this whole Sentry initialization:

```ts
if (process.env.NODE_ENV === "production") {
  Sentry.init(...);
}
```

Reply:
Cheers!